openrr-teleop/src/control_nodes_config.rs

```diff
@@ -56,20 +56,23 @@ impl ControlNodesConfig {
         ik_solver_with_chain_map: &HashMap<String, Arc<IkSolverWithChain>>,
         move_base: Option<Arc<dyn MoveBase>>,
         joints_poses: Vec<JointsPose>,
-    ) -> Vec<Box<dyn ControlNode>> {
-        let mut nodes: Vec<Box<dyn ControlNode>> = vec![];
+    ) -> Vec<Arc<Box<dyn ControlNode>>> {
```

Comment:
```suggestion
    ) -> Vec<Arc<dyn ControlNode>> {
```

Double boxing here is probably not necessary.

Reply:
https://github.com/openrr/openrr/pull/448/checks?check_run_id=3550417379#step:7:611

I couldn't solve the error...

Reply:
I guess that error can be solved by adding auto_impl(Arc) to ControlNode.

```diff
- #[auto_impl(Box)]
+ #[auto_impl(Box, Arc)]
  pub trait ControlNode: Send + Sync {
```

Reply:
https://github.com/openrr/openrr/pull/448/checks?check_run_id=3551247005#step:7:614
https://github.com/openrr/openrr/pull/448/commits/accf9f940470b90e92ef703a7e3437e45c5f2123

Reply:
Done